packages/gatsby/src/redux/reducers/jobsv2.ts

```diff
@@ -16,7 +23,9 @@ module.exports = (
 
     case `END_JOB_V2`: {
       const { jobContentDigest, result } = action.payload
-      const { job } = state.incomplete.get(jobContentDigest)
+      const { job } = state.incomplete.get(jobContentDigest) as IGatsbyJobV2
+
+      if (!job) return state
```

Comment:
Could we go for this syntax instead :)
```suggestion
      if (!job) {
        return state
      }
```

Reply:
Done!